docs/Search.md

```diff
@@ -0,0 +1,27 @@
+# Search
+
+Chatterino has Search Popup which lets you quickly find messages in the chat. You can open it with the `Ctrl+F` shortcut.
+
+## Search filters
+
+You can use several search filters to narrow down your search:
+
+- `from:<username>` - shows messages only from certain username. Can be multiple usernames separated with a comma (added in 2.1.4)
+- `has:<flags>` - shows messages only containing specified elements, list of flags (added in 2.1.4):
+    - `link` - filters messages with links
+- `in:<channel>` - (useful in `#/mentions` channel) shows messages that originate from certain channels (added in 2.1.4)
```

Comment:
```suggestion
- `in:<channel>` - (useful in `#/mentions` channel) shows messages that originate from certain channels (added in 2.3.0)
```

Maybe worth mentioning the option of comma seperated values here too? Or possibly once for all filters?

Reply:
I agree I've been a little inconsistent with the comma separation, I'll just make a note that all predicates can have comma-separated values :+1:

Reply:
Both resolved in fc3f0c2